editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -179,7 +208,12 @@ const Root = () => {
     return function cleanup() {
       map?.removeEventListener("geojsonChange", geojsonChangeHandler);
     };
-  }, [setFeatures]);
+  }, [setFeatures, addFeature]);
+
+  const rootError: string =
+    (errors.min && `You must provide at least ${schema.min} response(s)`) ||
+    (errors.max && `You can provide at most ${schema.max} response(s)`) ||
+    "";
```

Comment:
I've copied this verbatim from the List component - I'm not sure the concept of "response(s)" really holds up so well in this context?

Reply:
I think we'll want this language to be something like `"You must plot at least ${schema.min} ${schema.name} on the map"` - but very easy to fix forward !

Reply:
Yep this is a lot better! Will fix-forward in next PR 👍